x-pack/platform/plugins/private/reporting/public/management/components/report_schedules_table.tsx

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Fragment, default as React, useCallback, useState } from 'react';
+import {
+  EuiAvatar,
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHealth,
+  EuiIconTip,
+  EuiLink,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import moment from 'moment';
+import { orderBy } from 'lodash';
+import { stringify } from 'query-string';
+import { REPORTING_REDIRECT_APP, buildKibanaPath } from '@kbn/reporting-common';
+import type { ScheduledReportApiJSON, BaseParamsV2 } from '@kbn/reporting-common/types';
+import { ListingPropsInternal } from '..';
+import {
+  guessAppIconTypeFromObjectType,
+  getDisplayNameFromObjectType,
+  transformScheduledReport,
+} from '../utils';
+import { useGetScheduledList } from '../hooks/use_get_scheduled_list';
+import { prettyPrintJobType } from '../../../common/job_utils';
+import { ReportScheduleIndicator } from './report_schedule_indicator';
+import { useBulkDisable } from '../hooks/use_bulk_disable';
+import { NO_CREATED_REPORTS_DESCRIPTION } from '../../translations';
+import { ScheduledReportFlyout } from './scheduled_report_flyout';
+import { TruncatedTitle } from './truncated_title';
+import { DisableReportConfirmationModal } from './disable_report_confirmation_modal';
+
+interface QueryParams {
+  index: number;
+  size: number;
+}
+
+export const ReportSchedulesTable = (props: ListingPropsInternal) => {
+  const { http, toasts } = props;
+  const [selectedReport, setSelectedReport] = useState<ScheduledReportApiJSON | null>(null);
+  const [configFlyOut, setConfigFlyOut] = useState<boolean>(false);
+  const [disableFlyOut, setDisableFlyOut] = useState<boolean>(false);
+  const [queryParams, setQueryParams] = useState<QueryParams>({
+    index: 1,
+    size: 10,
+  });
+  const { data: scheduledList, isLoading } = useGetScheduledList({
+    http,
+    ...queryParams,
+  });
+
+  const { mutateAsync: bulkDisableScheduledReports } = useBulkDisable({
+    http,
+    toasts,
+  });
+
+  const sortedList = orderBy(scheduledList?.data || [], ['created_at'], ['desc']);
+
+  const tableColumns: Array<EuiBasicTableColumn<ScheduledReportApiJSON>> = [
+    {
+      field: 'payload.objectType',
+      name: i18n.translate('xpack.reporting.schedules.tableColumns.typeTitle', {
+        defaultMessage: 'Type',
+      }),
+      width: '5%',
+      render: (_objectType: string) => (
```

Comment:
Usually a variable that begins with a `_` means that it is unused, or needs to avoid shadowing a variable with the same name. Since neither is happening here, we don't need the `_`.